logging/tests/unit/test__gax.py

```diff
@@ -40,6 +40,7 @@ class _Base(object):
     PROJECT = 'PROJECT'
     PROJECT_PATH = 'projects/%s' % (PROJECT,)
     FILTER = 'logName:syslog AND severity>=ERROR'
+    UNIQUE_WRITER_IDENTITY = True
```

Comment:
How necessary is it to make this a constant? It's easy enough to just type the literals `True` and `False`?

Reply:
I guess just to be consistent with the other parameters, I'll keep it like that.

Reply:
The other parameters have meaning, `True` is a literal with only 1 bit of "entropy".